### DISCUSSION TITLE
Multiple font sizes

### DISCUSSION THREAD

**Discussion by ghost** [emilk/egui#954] [Q&A] [answered]
Hi,
I need multiple font sizes. But it seems that I can only have 5 different text styles for each frame. But I need more than that. Is there a way to set an arbitrary font size?

**Comment by BctfN0HUK7Yg** [comment]
https://docs.rs/egui/0.15.0/egui/struct.Ui.html#method.scope

**Reply by ghost** [reply]
How would this work?  Context.set_fonts(...) is still applied in the next frame, isn't it?

**Comment by manokara** [comment, marked as answer]
There has been some discussion on that in #56 that arbitrary sizes could be allowed by using SDF rendering, but it has been considered low priority, and there could be some performance tradeoffs in certain scenarios. So I guess for now you're going to have to use `scope()` and juggle `FontDefinition`s.
